torchgeo/datamodules/skippd.py

```diff
@@ -0,0 +1,52 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+"""SKy Images and Photovoltaic Power Dataset (SKIPP'D) datamodule."""
+
+from typing import Any
+
+import numpy as np
+from sklearn.model_selection import train_test_split
```

Comment:
We should avoid using sklearn, we're thinking about removing it. Instead use `.utils.dataset_split`